lib/decorators/activity-log.js

```diff
@@ -1,6 +1,24 @@
+const { get } = require('lodash');
 const Cacheable = require('./cacheable');
 const { autoResolved } = require('../flow/status');
 
+function getItemStatus(item) {
+  return item.eventName.split(':').pop();
+}
+
+function getStatusChange(item) {
+  return get(item, 'event.meta.payload.status');
+}
+
+function isDeadlineExtension(item) {
+  return !!get(item, 'event.meta.payload.data.extended');
+}
+
+function filterStatusChanges(item) {
+  const [type, , status] = item.eventName.split(':');
+  return (type === 'status' && status !== 'resubmitted') || isDeadlineExtension(item);
```

Comment:
What's the logic behind removing `resubmitted`?

Reply:
resubmitted autoforwards, but doesn't have a meta.payload.status - we display the thing that it autoforwards too eg with licensing. If we leave it in there are 2 logs for the same event